### DISCUSSION TITLE
Support for CommonJS

### DISCUSSION THREAD

**Discussion by JurajKavka** [igordanchenko/yet-another-react-lightbox#188] [Q&A] [answered]
I have question if it is possible to build this package with support for CommonJS. We have custom SSR that is quite obsolete but still works and when I call the service it fails with error...

```
 require() of ES Module /XYZ/node_modules/yet-another-react-lightbox/dist/index.js from /XYZ/node_modules/XYZ/PhotoGallery/SimplePhotoGallery.js not supported.
Instead change the require of index.js in /XYZ/PhotoGallery/SimplePhotoGallery.js to a dynamic import() which is available in all CommonJS modules.
```

Thank you.

**Comment by igordanchenko** [comment, marked as answer]
Unfortunately, no. This package is distributed as ESM-only as it's meant to be used in projects with modern bundlers that can handle ESM-only dependencies. You can probably build your own CJS distro from source if you are unable to modernize your build toolchain.

**Reply by JurajKavka** [reply]
OK I understand, thank you for the response and the great library!
